ui/components/app/wallet-overview/hooks/useHandleSendNonEvm.ts

```diff
@@ -26,7 +25,10 @@ import { getMultichainNativeAssetType } from '../../../../selectors/assets';
  * @returns A function that triggers the send flow for non-EVM accounts.
  */
 export const useHandleSendNonEvm = (caipAssetType?: CaipAssetType) => {
-  const nativeAssetType = useMultichainSelector(getMultichainNativeAssetType);
+  const selectedNetwork = useSelector(
+    getSelectedMultichainNetworkConfiguration,
+  );
+  const { nativeCurrency } = selectedNetwork;
```

Comment:
We could refactor this a bit? Unless you prefer doing it in 2 steps?

```suggestion
  const { nativeCurrency } = useSelector(
    getSelectedMultichainNetworkConfiguration,
  );
```

Reply:
Done in https://github.com/MetaMask/metamask-extension/pull/31702/commits/171edcb7364d3b784770a5ebbf2aa53619047727